src/bibx/__init__.py

```diff
@@ -57,6 +57,12 @@ def read_any(file: TextIO) -> Collection:
     Tries to read a file with the supported formats.
     """
     for handler in (read_wos, read_scopus_ris, read_scopus_bib):
-        with suppress(BibXError):
+        try:
             return handler(file)
-    raise ValueError("Unsuported file type")
+        except BibXError as e:
+            print(f"Error: {e}")
```

Comment:
Can we use logger.debug, instead of `pring`. https://docs.python.org/3/library/logging.html